baseband/data/sample_gsb_phased.timestamp

```diff
@@ -0,0 +1,10 @@
+2013 07 28 02 53 55 0.517535 2013 07 28 02 53 55 0.324108800 9995 3 
+2013 07 28 02 53 55 0.769434 2013 07 28 02 53 55 0.575767040 9996 4 
+2013 07 28 02 53 56 0.020854 2013 07 28 02 53 55 0.827425280 9997 5 
+2013 07 28 02 53 56 0.272643 2013 07 28 02 53 56 0.079083520 9998 6 
+2013 07 28 02 53 56 0.524170 2013 07 28 02 53 56 0.330741760 9999 7 
+2013 07 28 02 53 56 0.775977 2013 07 28 02 53 56 0.582400000 10000 0 
+2013 07 28 02 53 57 0.027501 2013 07 28 02 53 56 0.834058240 10001 1 
+2013 07 28 02 53 57 0.279274 2013 07 28 02 53 57 0.085716480 10002 2 
+2013 07 28 02 53 57 0.530805 2013 07 28 02 53 57 0.337374720 10003 3 
+2013 07 28 02 53 57 0.782605 2013 07 28 02 53 57 0.589032960 10004 4 
```

Comment:
Timestamp files don't have newlines (though it's hard to tell with phased, since all the ones I have access to end in incomplete headers), so I'll keep this as is.

Reply:
That suggests perhaps we should put in an incomplete header - that will test `stop_time` better as well.